app/src/main/java/com/sopt/now/ui/home/HomeFragment.kt

```diff
@@ -30,34 +37,71 @@ class HomeFragment(private val user: User) : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        userId = activity?.intent?.getStringExtra(USER_ID)
+
+        getFriendData()
+        getUserData(userId)
 
         multiAdapter = MultiAdapter()
+
         binding.rvHomeFriends.apply {
             adapter = multiAdapter
             layoutManager = LinearLayoutManager(requireContext())
         }
 
-        combineList()
     }
 
-    private fun loadFriendList(): MutableList<Friend> {
-        val friendList: MutableList<Friend> = mutableListOf()
-        friendList.addAll(viewModel.mockFriendList)
-
-        return friendList
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
     }
 
-    private fun combineList() {
-        val combinedList = mutableListOf<Any>().apply {
-            add(user.copy())
-            addAll(loadFriendList())
-        }
+    var mockFriendList = mutableListOf<Friend>()
+    private fun getFriendData() {
+        ServicePool.friendService.getFriend(2).enqueue(object : Callback<ResponseFriendDto> {
+            override fun onResponse(
+                call: Call<ResponseFriendDto>,
+                response: Response<ResponseFriendDto>
+            ) {
+                if(response.isSuccessful) {
+                    val friend = response.body()?.data
 
-        multiAdapter.setItemList(combinedList)
+                    friend?.forEach { friendData ->
+                        mockFriendList.add(
+                            Friend(friendData.avatar, friendData.first_name, friendData.email)
+                        )
+                    }
+
+                    multiAdapter.setFriendList(mockFriendList)
+                } else {
+                    Log.d("HomeViewModel", "Failed to load user data: ${response.message()}")
```

Comment:
nit : 이런 로그들이 있으면 나중에 디버깅할 때 보기 좋을 것 같아요 👍 에러라면 `Log.e`로 에러를 찍어도 될 것 같아요.

```suggestion
                    Log.e("HomeViewModel", "Failed to load user data: ${response.message()}")
```

Reply:
넵 수정하겠습니다! 👍